dash/webapp/next.config.js

```diff
@@ -1,11 +1,4 @@
 module.exports = {
-  exportPathMap: async function (defaultPathMap) {
-    return {
-      '/': { page: '/' },
```

Comment:
This used to be for the output, so it would follow this file structure we have in the S3 bucket, but with the move to Next14 I:
> - Removed export task as build now creates the output folder (bc I added it to the config as per [migration docs](https://nextjs.org/docs/app/building-your-application/upgrading/version-14#v14-summary))
> - Set [trailingSlash](https://nextjs.org/docs/pages/api-reference/next-config-js/trailingSlash) to match the existing output structure as exportPathMap is deprecated.

Hope that answers it?